public/data/en/fields.json

```diff
@@ -537,7 +537,7 @@
     "label": "I do not have a street address.",
     "help_text": false,
     "error_msg": false,
-    "instructions": "",
+    "instructions": "Please only select the checkbox below if it is applicable to you.",
```

Comment:
Revert this - it will be overwritten when we pull in all the json from the CMS. I'll tag Susan from content to add this on the CMS side.